index.ts

```diff
@@ -931,6 +933,41 @@ commander
     });
   });
 
+/**
+ * Lists the apps scripts
```

Comment:
Can you give more detail example? Answer these questions:
- Does it return all my scripts? (no)
- What does the output look like (name + id) (Give an `@example`)